server/controllers/request.controller.js

```diff
@@ -0,0 +1,61 @@
+const RequestInfo = require('../models/request.model.js');
+
+exports.getRequests = async (req, res) => {
```

Comment:
You should utilise express-async-handler because that is also used in our other controllers and for consistency that makes more sense, also with that change we wouldn't need to do try/catch because express async handler will catch the errors and send a json response.